R/pvclust.R

```diff
@@ -73,7 +73,8 @@ na_locf <- function(x, first_na_value = 0, recursive = TRUE, ...) {
   } else {
     # If the first observation is NA, fill it with "first_na_value"
     if (x_na[1]) x[1] <- first_na_value
-
+    x_na[1] <- F
```

Comment:
Foe the future, In code it's better to use
FALSE
And not just 
F

Since a user could by accident do
F <- T


For analysis it doesn't matter much. For R packages it matters more.